### PR TITLE
WID-406-Abandoned-cart-emails commit message

### PR DESCRIPTION
**Pull request by sitikanthasezzle**
Instructions for the abandoned cart emails

### CODE REVIEW

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget-modal.html [hunk 1]
```diff
@@ -0,0 +1,417 @@
+
+<!-- If you wish to use the regular Sezzle Modal, you will need to replace: -->
```
Comment (aholdahl) [inline src/abandoned-cart-email-widget/abandoned-cart-email-widget-modal.html:2]: Did you test? Is it actually possible to add a modal to an email?

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+<!-- 
+
+The Sezzle Shopify abandoned cart email widget is a lightweight version of our static widget.It suggests the merchants to use Sezzle widget in their brand’s email footers and abandoned cart emails.
+It is mainly self-contained widget version for implementation without connection to Sezzle’s server, which can be copied to the merchant theme via Github
+
+Log into Shopify Admin
+Go to Settings > Notifications > Abandoned Checkout
+Click the <> (Show HTML) button within the description or wthin footer tag
```
Comment (aholdahl) [inline src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:9]: The <> (Show HTML) button doesn't seem to exist on this page. It just opens to Liquid editor.

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Paste the static widget at the top or bottom of the code window, depending on where you would like the widget to appear
```
Comment (aholdahl) [inline src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:10]: This should probably be edited so it reads "Paste the below code snippet where you would like the widget to appear." Probably also a good idea to include a suggestion, such as between the products table and the "row footer" table.

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+Click Save
+
+* If your product page is a dark background, replace the img src URL with "https://media.sezzle.com/branding/2.0/Sezzle_Logo_FullColor_WhiteWM.svg"
```
Comment (aholdahl) [inline src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:13]: Is it actually possible to have a dark mode abandoned cart email? Or is this line unnecessary?

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+-->
+<p class="sezzle-shopify-buy-static-widget">Buy now, pay later in 4 interest-free payments with <img class="sezzle-logo" src="https://media.sezzle.com/branding/2.0/Sezzle_Logo_FullColor.svg" alt="Sezzle logo" style="display: inline-block; height: 18px!important; margin: 0px 0px -3px 0px;"> <a class="sezzle-info-link" href="https://sezzle.com/how-it-works" target="_blank" style="text-decoration: none; border: none; padding: 0px 5px;" rel="noopener noreferrer">&#9432;</a></p>
+<script type="text/javascript">
```
Comment (aholdahl) [inline src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:16]: Stray opening script tag needs removed.

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
+
+<!-- 
+
+The Sezzle Shopify abandoned cart email widget is a lightweight version of our static widget.It suggests the merchants to use Sezzle widget in their brand’s email footers and abandoned cart emails.
```
Comment (aholdahl) [inline src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:4]: This doesn't necessarily need to be exclusive to Shopify though. Perhaps we can tailor the language to say the below instructions were written for Shopify but the code can be modified for other platforms as needed?

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html (hunk 4 above)
Comment (sitikanthasezzle) [reply src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:13]: yes  possible  if the background of abandoned cart email page is dark @aholdahl

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html (hunk 5 above)
Comment (sitikanthasezzle) [reply src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:16]: @aholdahl  will remove the tag in the next merge request... thanks

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html (hunk 2 above)
Comment (sitikanthasezzle) [reply src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:9]: made changes and removed it

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html (hunk 3 above)
Comment (sitikanthasezzle) [reply src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:10]: fixed

File: src/abandoned-cart-email-widget/abandoned-cart-email-widget.html (hunk 5 above)
Comment (sitikanthasezzle) [reply src/abandoned-cart-email-widget/abandoned-cart-email-widget.html:16]: fixed

**Pull request merged**